test/builder/test_suite.go

```diff
@@ -223,8 +223,14 @@ func (s *TestSuite) init(crdPaths []string) {
 	s.flags = flags
 
 	if s.flags.IntegrationTestsEnabled {
+		var assetsDir string
+		if _, exists := os.LookupEnv("KUBEBUILDER_ASSETS"); !exists {
+			assetsDir = filepath.Join(testutil.GetRootDirOrDie(), "hack/tools/bin/")
+		}
+
 		s.envTest = envtest.Environment{
-			CRDDirectoryPaths: crdPaths,
+			CRDDirectoryPaths:     crdPaths,
+			BinaryAssetsDirectory: assetsDir,
```

Comment:
Given that this field can be overridden by setting KUBEBUILDER_ASSETS, perhaps just set it here without checking the env var existence above?